Extension/background.js

```diff
@@ -19,6 +19,23 @@ function snagLink(pageDetails) {
   copyToClipboard('[' + pageDetails.issueNumber + ' - ' + pageDetails.title + '](http://jiraprod.agfahealthcare.com/browse/' + pageDetails.issueNumber + ')');
 }
 
+function snagHtmlLink(pageDetails) {
+  var anchor = document.getElementById('htmlLink');
+  anchor.setAttribute("href", "http://jiraprod.agfahealthcare.com/browse/" + pageDetails.issueNumber);
+  anchor.innerHTML = pageDetails.issueNumber + ' - ' + pageDetails.title;
+
+  var div = document.getElementById('htmlLinkDiv');
+  div.style.display = 'block';
+  var range = document.createRange();
+  var selection = window.getSelection();
+  range.selectNodeContents(div);
+  selection.removeAllRanges();
+  selection.addRange( range );

```

Comment:
While I'm nitting (see below), I find the "declare range, declare selection, do something with range, do something with selection, do something with selection and range" pattern slightly harder to read than, e.g. "declare range, do something with range, declare selection, do something with selection, do something with range and selection". The short version of that is "How do you feel about swapping lines 30 and 31? Not a huge deal.